.github/workflows/assimp.yml

```diff
@@ -17,17 +17,34 @@ jobs:
       fail-fast: false
       matrix:
         env:
-          - os: ubuntu-latest
+          - os: ubuntu-22.04
             name: Linux
             nuke_invoke: ./build.sh
             extras: |
-              sudo apt-get update
-              sudo apt-get install -y xorg-dev 
-          - os: windows-latest
+              sudo tee /etc/apt/sources.list << EOF
+              deb [arch=amd64] http://archive.ubuntu.com/ubuntu jammy main universe
+              deb [arch=amd64] http://archive.ubuntu.com/ubuntu jammy-backports main universe
+              deb [arch=amd64] http://archive.ubuntu.com/ubuntu jammy-security main universe
+              deb [arch=amd64] http://archive.ubuntu.com/ubuntu jammy-updates main universe
+              deb [arch=arm64,armhf] http://ports.ubuntu.com jammy main universe
+              deb [arch=arm64,armhf] http://ports.ubuntu.com jammy-backports main universe
+              deb [arch=arm64,armhf] http://ports.ubuntu.com jammy-security main universe
+              deb [arch=arm64,armhf] http://ports.ubuntu.com jammy-updates main universe
+              deb [arch=amd64] http://security.ubuntu.com/ubuntu jammy main universe
+              deb [arch=amd64] http://security.ubuntu.com/ubuntu jammy-backports main universe
+              deb [arch=amd64] http://security.ubuntu.com/ubuntu jammy-security main universe
+              deb [arch=amd64] http://security.ubuntu.com/ubuntu jammy-updates main universe
+              EOF
+              sudo dpkg --add-architecture arm64
+              sudo dpkg --add-architecture armhf
+              sudo apt update
+              sudo apt install -y gcc-aarch64-linux-gnu gcc-arm-linux-gnueabihf
+              sudo apt install -y g++-aarch64-linux-gnu g++-arm-linux-gnueabihf
```

Comment:
i'd prefer if any custom sources are commented for why they exist, i did a poor job of this when i did this last, and it caused confusion, so lets try to not cause more in the future

Reply:
In this case it's all just the official sources needed for an `amd64`/`armhf`/`arm64` multiarch setup in 22.04. I can add a comment to that effect.